<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Socketset</class>
 <widget class="QWidget" name="Socketset">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>通道1(配置平台)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>通道2(数据平台)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>通道3(数据推送)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_1_ip">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2_ip">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3_ip">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_1_port">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2_port">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>100</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3_port">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>160</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_1">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>79</width>
     <height>23</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>TCP</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>UDP</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MQTT</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>100</y>
     <width>79</width>
     <height>23</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>TCP</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>UDP</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MQTT</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>160</y>
     <width>79</width>
     <height>23</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>TCP</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>UDP</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MQTT</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_save">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>200</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ip">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>IP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_port">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>端口</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_protocol">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>协议</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
